docs/data/toolpad/getting-started/quickstart.md

```diff
@@ -23,7 +23,7 @@ Make sure to [install Node.js](https://nodejs.org/en) on your system.
 
 Interactive walkthrough of the app building process:
 
-<div style="position: relative; padding-bottom: calc(54.79166666666667% + 41px); height: 0;"><iframe src="https://demo.arcade.software/BhTsUpHyAYGApTZ2mFNj?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;" title="Default page | Toolpad editor"></iframe></div>
+<div class="arcade"><iframe src="https://demo.arcade.software/BhTsUpHyAYGApTZ2mFNj?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;" title="Default page | Toolpad editor"></iframe></div>
```

Comment:
Something is broken with this iframe, there is a scrollbar

<img width="395" alt="Screenshot 2023-05-20 at 00 13 45" src="https://github.com/mui/mui-toolpad/assets/3165635/3135be39-3bb6-47c3-84d5-857d6af5af6d">

Reply:
[Updated](https://github.com/mui/mui-toolpad/pull/2045/commits/568cb17c6acfc8c3444a11d9828e34c3b9008fc0) arcade, but the vertical scrollbar is still there. How can that be removed? It was recorded at 1440x796 (DPR 2.0)

Reply:
@prakhargupta1 the issue is with code, I didn't check why